test/test-helper.el

```diff
@@ -118,17 +118,34 @@ If SECTION is not supplied, use the value of
       (set-text-properties 0 (length result) nil result)
       result)))
 
-
-(defmacro magit-gh--with-temp-buffer (&rest forms)
```

Comment:
Do we not need this anymore?

Reply:
Never mind, I see that it moved here: https://github.com/astahlman/magit-gh-comments/blob/c150d4f123d4481c74e5a6caf04a5cfb6c3ae241/magit-gh-comments.el#L80